src/Header/Search/Search.test.jsx

```diff
@@ -73,36 +73,6 @@ describe("Search", () => {
 	});
 
 	describe("onSearching", () => {
-		it("should call searchSubmitHandler when the form is submitted", () => {
```

Comment:
Please can you re-instate these tests?

Reply:
Yes, will re-instate and try to make them work with hooks or write equivalents

Reply:
Updated these tests